Toolset/palettes/script editor/behaviors/revseeditorbehavior.livecodescript

```diff
@@ -951,7 +951,7 @@ command scriptComment
    # OK-2009-03-03 : Bug 7537 - Reformat the handler after commenting, this may be flawed,
    # but the proper solution is rather time-consuming.
```

Comment:
well it doesn't seem the correct thing if you are only commenting a few lines from a many thousand line script... ideally it would only format the selected lines